projects/igniteui-angular/src/lib/drop-down/drop-down-item.component.ts

```diff
@@ -205,7 +205,7 @@ export class IgxDropDownItemBase {
             if (focusedItem) {
```

Comment:
Shouldn't this respect `allowItemsFocus` and not call the native focus if not enabled?

Reply:
Same applies for the `isFocused` setter TBH.

Reply:
fixed